autogpt/auto_gpt_workspace/instance_history.txt

```diff
@@ -0,0 +1,2 @@
+Auto-GPT instance history:
```

Comment:
Should this be here?

Reply:
uhm, no idea how that happened, one second...